pkg/codeowners/owners.go

```diff
@@ -35,6 +40,34 @@ func NewFromPath(path string) ([]Entry, error) {
 	return ParseCodeowners(r), nil
 }
 
+func checkMultipleCodeownersFiles(dir string) error {
+	var codeownerFileCount int
+	for _, p := range []string{".", "docs", ".github"} {
```

Comment:
I think it might be cleaner to have these "constant" string values be defined in a global array on top of this file, somewhere between line 15 and 17:

i.e.
```
var acceptedCodeownerLocations = [3]string{".", "docs", ".github"}
```

Reply:
I changed the code to your suggestion so that the constants are only used in one spot now. Generally I agree that it would be cleaner, but this is an external project and I want to make my PR the simplest possible to increase the chance it will get merged in, so I'm hesitant to move these to a constant string value. 

If you feel strongly about it/think it will help it get merged I'm down to do it

Reply:
oh yea, sounds good. no worries.